### PR TITLE
Expose /contribute for L10n (Fixes #9151)

### LINKED ISSUE #9151: Expose /contribute for L10n
## Description

The redesigned [contribute page](https://www.mozilla.org/en-US/contribute/) has yet to be exposed for L10n. The template also needs converting to Fluent.

---

#### Success Criteria
- [ ] Strings for new template are available in Pontoon.

#### Approvals
- [ ] The Accountable
- [ ] Brand - design?
- [ ] Brand - copy?
- [ ] Team Lead
- [ ] Others?

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Adds `mozorg/contribute.ftl`
- Converts `/contribute/` page to use Fluent IDs.
- Updates view tests.

## Issue / Bugzilla link
#9151

## Testing
- [x] Did I miss any strings?
- [x] Did I miss any brand names?

### CODE REVIEW

File: bedrock/mozorg/templates/mozorg/contribute/contribute-2020.html [hunk 1]
```diff
@@ -89,13 +83,11 @@ <h2 class="mzp-c-card-title">Contribute to the Mozilla codebase</h2>
...
-              <div class="mzp-c-card-tag">Individual and event organization</div>
+              <div class="mzp-c-card-tag contribute-card-tag-bold">{{ ftl('contribute-tag-support') }}</div>
+              <div class="mzp-c-card-tag">{{ ftl('contribute-tag-individual-event') }}</div>
             </div>
-            <h2 class="mzp-c-card-title">Individual and event organization</h2>
-            <p class="mzp-c-card-desc">Help make Mozilla’s products easy to
-              use. Answer user's "help" questions as part of the Mozilla Support
-              Community forums.</p>
+            <h2 class="mzp-c-card-title">{{ ftl('contribute-tag-individual-event') }}</h2>
+            <p class="mzp-c-card-desc">{{ ftl('contribute-task-support-desc') }}.</p>
```
Comment (craigcook) [inline bedrock/mozorg/templates/mozorg/contribute/contribute-2020.html:90]: Stray `.` at the end here.

File: l10n/en/mozorg/contribute.ftl [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+contribute-tag-technical = Technical
+contribute-tag-technical-non-technical = Technical and Non-technical
+contribute-tag-individual-event = Individual and event organization
+contribute-tag-individual = Individual
+contribute-tag-support = Support
+contribute-task-translate-heading = Translate content
+contribute-task-translate-desc = The internet is only global if it’s understood everywhere. Help us translate { -brand-name-mozilla } products and websites into your local language.
+contribute-task-mozilla-codebase-heading = Contribute to the { -brand-name-mozilla } codebase
+contribute-task-mozilla-codebase-desc = Actively improve { -brand-name-mozilla } products by contributing to a variety of development opportunities.
+contribute-task-support-desc = Help make { -brand-name-mozilla } products easy to use. Answer user’s "help" questions as part of the { -brand-name-mozilla } Support Community forums.
```
Comment (craigcook) [inline l10n/en/mozorg/contribute.ftl:26]: Should probably use curly quotes around `help`

**Review by craigcook — CHANGES_REQUESTED** [review]
Just two teeny nits. r+wc 🧈

File: l10n/en/mozorg/contribute.ftl (hunk 2 above)
Comment (craigcook) [reply l10n/en/mozorg/contribute.ftl:26]: Also, I think this should be `users’` to make it plural, though it's one of those weird possessives with the hanging apostrophe. Or we could try `answer people’s “help” questions` if we think `users’` is too clunky.

**Comment by alexgibson** [comment]
Thanks @craigcook updated

**Review by craigcook — APPROVED** [review]

**Pull request merged**